cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -98,6 +98,9 @@ metadata:
   # should be in the namespace of the operator
   namespace: rook-ceph # namespace:operator
 data:
+  # The logging level for the operator: INFO | DEBUG
+  ROOK_LOG_LEVEL: "INFO"
```

Comment:
Let's remove the ROOK_LOG_LEVEL from the env vars so that this becomes the preferred place to specify it. It's confusing if the example has it in both places.

Reply:
removed